.github/workflows/ci.yml

```diff
@@ -889,6 +893,10 @@ jobs:
         shell: powershell
         run: |
           $versionV = "${{ github.event.release.tag_name }}"
+          if (!$versionV) {
+            # Signing was triggered by workflow dispatch.
```

Comment:
Maybe explain why this fake version is okay? We using this for testing the workflow.

Reply:
done.